src/main/java/gui/events/CameraShotCreationEvent.java

```diff
@@ -0,0 +1,42 @@
+package gui.events;
+
+import lombok.Getter;
+
+import java.util.List;
+
+/**
+ * Class that represents a director shot creation event.
+ * @author martijn
+ */
+public class CameraShotCreationEvent extends ModalEvent {
+
+    @Getter
+    private String shotName;
+    @Getter
+    private String shotDescription;
+    @Getter
+    private List<Integer> camerasInShot;
```

Comment:
This also refers to the shots in the timeline, not the DirectorShot. The javadoc is incorrect. I will change that.